extractor/src/main/java/org/schabi/newpipe/extractor/MediaFormat.java

```diff
@@ -121,6 +132,12 @@ public static MediaFormat getFromMimeType(final String mimeType) {
                 .orElse(null);
     }
 
+    public static List<MediaFormat> getAllFromMimeType(final String mimeType) {
+        return Arrays.stream(MediaFormat.values())
+                .filter(mediaFormat -> mediaFormat.mimeType.equals(mimeType))
+                .collect(Collectors.toList());
+    }
```

Comment:
This method should be documented and annotations should be added.

Also, should we make the list returned unmodifiable?

Reply:
I don't think there is any good reason to make the list unmodifiable. The user of the interface might want to use that List, e.g. reduce the entries.